### PR TITLE
Grammar, Spelling, Formatting

### PR DESCRIPTION
**Pull request by ashnazg**
(cosmetic changes only)

### CODE REVIEW

**Review by garak — APPROVED** [review]

**Comment by ashnazg** [comment]
ping @cseufert... GH won't let me choose you as a reviewer/assignee.

**Review by samdark — APPROVED** [review]

File: proposed/clock.md [hunk 1]
```diff
@@ -46,7 +44,7 @@ $timestamp = $clock->now()->getTimestamp();
 ## 2.1 ClockInterface
 
 The clock interface defines the most basic operations to read the current time and date from the clock. 
-It MUST return the time as a `DateTimeImmutable`.
+It MUST return the time as a `\DateTimeImmutable`.
 
 ~~~php
```
Comment (cseufert) [inline proposed/clock.md:49]: While we are tidying up things, we should use the ``` code block rather than mixing between the tilde and the backtick.

**Review by cseufert — COMMENTED** [review]
Looks, good overall, however the `ClockInterface` class has no explicit namespace, should this be done for consistency?

**Comment by heiglandreas** [comment]
I'm not really in favour of using the backslash to denote the root-namespace. In code I'd rather explicitly use `use`-statements also for functions etc. But in a plain text for documentation purposes I'd rather leave it out for readability purposes. 

But *if*  we want to use the backslash, then we need to do it consistently everywhere. There where a few parts where that then needs to be added as well :wink:

**Review by samdark — APPROVED** [review]

**Review by BafS — APPROVED** [review]

**Comment by ashnazg** [comment]
@cseufert ping... Github shows you still active in general, so reaching out to you here rather than just on Discord to confirm you're still interested in this PSR... ?

**Comment by cseufert** [comment]
@ashnazg sorry so busy it has slipped off my radar, however, I am still keen to complete this PSR

**Comment by cseufert** [comment]
> I'm not really in favour of using the backslash to denote the root-namespace. In code I'd rather explicitly use `use`-statements also for functions etc. But in a plain text for documentation purposes I'd rather leave it out for readability purposes.
> 
> But _if_ we want to use the backslash, then we need to do it consistently everywhere. There where a few parts where that then needs to be added as well wink

I think that is fair for code blocks, however for mentioning classes, and functions in the body of the spec, I think it is clearer to explicitly use the full path.

**Review by cseufert — APPROVED** [review]
Looks great, lets's get this merged

File: proposed/clock-meta.md [hunk 2]
```diff
@@ -2,29 +2,29 @@
...
 
 ## 2. Why Bother?
 
-There are currently a few libraries that provide this functionality, however there is no interopability between these different libraries, as they ship with their own clock interfaces. 
+There are currently a few libraries that provide this functionality. However, there is no interoperability between these different libraries, as they ship with their own clock interfaces.
 
-Symfony provides a package called `symfony/phpunit-bridge` that has a `Symfony\Bridge\PhpUnit\ClockMock` class, which allows mocking PHP's built-in time and date functions, however this does not solve mocking calls to `new \DateTimeImmutable()`. It also does not fully mock time when called from other libraries that rely on the system time.
+Symfony provides a package called `symfony/phpunit-bridge` that has a `Symfony\Bridge\PhpUnit\ClockMock` class, which allows mocking PHP's built-in time and date functions. However, this does not solve mocking calls to `new \DateTimeImmutable()`. It also does not fully mock time when called from other libraries that rely on the system time.
```
Comment (ashnazg) [inline proposed/clock-meta.md:15]: I can also add leading backslash in namespace class names like seen here, if we want it.

Comment (cseufert) [reply proposed/clock-meta.md:15]: For consistency, it makes sense

**Comment by ashnazg** [comment]
@cseufert, as Editor, you can merge these, right?

**Comment by ashnazg** [comment]
@Jean85 , hit this one too, please.

**Comment by Jean85** [comment]
Roger! :+1:

**Pull request merged**